### PR TITLE
Add preprocess and postprocess for additional trgs

### PR DESCRIPTION
**Pull request by bonlime**
Fixes https://github.com/albu/albumentations/issues/286
I'm not sure this is the best way to fix it, but I didn't find any shorter way to do that

### CODE REVIEW

**Comment by bonlime** [comment]
The build doesn't pass due to flake8 warnings "Line too long" but except that everything passes

**Comment by BloodAxe** [comment]
Hi. "Line too long" warning can be easily fixed by breaking long lines into two. Please show some respect to code-style ;)
Second, would you please add unit tests to ensure this change has expected behavior and does not change existing correct functionality?

**Comment by bonlime** [comment]
@BloodAxe
bump.

File: albumentations/core/composition.py [hunk 1]
```diff
@@ -225,28 +236,28 @@ def data_postprocessing(data_name, params, check_fn, filter_fn, convert_fn, data
...
+                                    check_validity=bool(params.get('remove_invisible', True)))
 
-    data = remove_label_fields_from_data(data_name, params.get('label_fields', []), data)
+    data = remove_label_fields_from_data(trg_name, params.get('label_fields', []), data)
     return data
 
 
-def data_preprocessing(data_name, params, check_fn, convert_fn, data):
+def data_preprocessing(trg_name, data_name, params, check_fn, convert_fn, data):
     if data_name not in data:
```
Comment (albu) [inline albumentations/core/composition.py]: please add trg_name to this assertion

**Comment by albu** [comment]
will be fixed in https://github.com/albu/albumentations/pull/319
sorry for that

**Pull request closed without merging**